draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -363,32 +363,31 @@
 
             <t>LSPs</t>
 
-            <t>Complete Sequence Number PDUs (CSNPs) and Partial Sequence
-            Number PDUs (PSNPs)</t>
+            <t>Complete Sequence Number PDUs (CSNPs) and PSNPs</t>
           </list>Implementations today may prioritize the reception of Hellos
         over LSPs and SNPs in order to prevent a burst of LSP updates from
         triggering an adjacency timeout which in turn would require additional
         LSPs to be updated.</t>
 
-        <t>SNPs serve to acknowledge or trigger the transmission of specified
+        <t>CSNPs and PSNPs serve to acknowledge or trigger the transmission of specified
         LSPs. On a point-to-point link, PSNPs acknowledge the receipt of one
-        or more LSPs. Because PSNPs (like all IS-IS PDUs) use TLVs in the
+        or more LSPs. Because PSNPs use TLVs in the
         body, it is possible to acknowledge multiple LSPs using a single PSNP.
         For this reason, <xref target="ISO10589"/> specifies a delay
         (partialSNPInterval) before sending a PSNP so that the number of PSNPs
         required to be sent is reduced. On receipt of a PSNP, the set of LSPs
         acknowledged by that PSNP can be marked so that they do not need to be
         retransmitted.</t>
 
-        <t>If a PSNP is dropped on reception, this has a significant impact as
+        <t>If a PSNP is dropped on reception, 
         the set of LSPs advertised in the PSNP cannot be marked as
-        acknowledged and this results in needless retransmissions which may
-        further delay transmission of other LSPs which have yet to be
+        acknowledged and this results in needless retransmissions that will
+        further delay transmission of other LSPs that have yet to be
         transmitted. It may also make it more likely that a receiver becomes
         overwhelmed by LSP transmissions.</t>
 
         <t>It is therefore RECOMMENDED that implementations prioritize the
-        receipt of Hellos and then SNPs over LSPs. Implementations MAY also prioritize IS-IS packets over other protocols either less critical or more chatty/bursty.</t>
+        receipt of Hellos and then SNPs over LSPs. Implementations MAY also prioritize IS-IS packets over other less critical protocols.</t>
```

Comment:
"It is therefore RECOMMENDED that implementatons apply three levels
of priority to incoming IS-IS PDUs:

   High: Hellos
   Medium: xSNPs
   Low: LSPs"

I think the mention of "less critical protocols" is unnecessarily provocative
and should be removed.

Reply:
I respectfully disagree.  IGPs should be prioritized above SNMP and telemetry.  IGPs need to be prioritized above BGP.

Reply:
I agree with you. 
But throwing in a single statement about "less critical protocols" introduces the question of what is the "correct" prioritization i.e., which protocols are critical, which are high, which are medium...
Do you want to include such a discussion in this document? If yes, then we need write more text. If not, then I think we are better off not saying anything.